app/views/layouts/components/_testimonial.html.erb

```diff
@@ -0,0 +1,31 @@
+<% case testimonial[:heading][:level] %>
+<% when :heading_one %>
```

Comment:
looks like some duplication :eyes: maybe we can create a `render/heading` partial which does this?

Reply:
This will be rectified in a future PR when handling of alignment in the `GetCaseStudy` use case and refactoring of current CSS is completed.